src/js/common/schemaform/helpers.js

```diff
@@ -533,3 +533,18 @@ export function getActiveChapters(formConfig, formData) {
 
   return _.uniq(expandedPageList.map(p => p.chapterKey).filter(key => !!key && key !== 'review'));
 }
+
+export function sanitizeForm(formData) {
+  try {
+    const suffixes = ['first', 'last', 'accountNumber', 'socialSecurityNumber', 'dateOfBirth'];
```

Comment:
should `vaFileNumber` also be included here? or any others?

Reply:
vaFileNumber is a good one to include, since that might be an SSN.